src/Nethermind/Nethermind.Synchronization.Test/E2ESyncTests.cs

```diff
@@ -172,12 +163,19 @@ private IContainer CreateNode(PrivateKey nodeKey, Action<IConfigProvider, ChainS
                 ))
                 .AddDecorator<ITestEnv, PostMergeTestEnv>()
                 ;
-
+        }
+        else
+        {
+            // So that any EIP after the merge is not activated.
+            DateTime mergeTime = new DateTime(2022, 9, 15, 13, 45, 0, DateTimeKind.Utc);
+            ManualTimestamper timestamper = new ManualTimestamper(mergeTime.AddDays(-1));
```

Comment:
Maybe abstract in `static ManualTimestamper.CreatePreMerge`?
Might be useful somewhere else in the future.

Reply:
Sure